docs/index.md

```diff
@@ -1,116 +1,30 @@
-# Research Computing User Guide
+# CU Research Computing User Guide
 
-Documentation covering the use of Research Computing resources.
+Welcome to CU Research Computing's (CURC's) user guide! CURC offers a variety of services encompassing High Performance Computing (HPC), cloud computing assistance, and storage solutions. Many of these services are available to users associated with CU Boulder, Anschutz Medical Campus (AMC), Colorado State University (CSU), and institutions who are members of the Rocky Mountain Advanced Computing Consortium (RMACC). 
```

Comment:
Grammar Change "institutions who are members" to "institutions that are members"